### DISCUSSION TITLE
[Enhancement] [table] [Component] [table] el-table树状结构selection列，希望增加一个“是否需要同时勾选子孙“行的可选属性

### DISCUSSION THREAD

**Discussion by wonderl17** [element-plus/element-plus#12163] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table`

## Description

el-table树状结构的selection列：
目前是选中or取消父数据都会自动同步到子孙数据;
希望能有一个属性配置为每行数据的选中or取消相互独立，不受祖先级数据关联。

### Additional comments

_(empty)_

**Comment by xiaolin089** [comment]
我这边刚好有个需求是反过来的，不允许父数据勾选，父数据相当于是个分组的标题，然后只允许分组里的子数据勾选，然后表格左上角全选，要选中所有子数据。
